client/app/scripts/liveblog-themes/module.js

```diff
@@ -3,6 +3,7 @@
 
     LiveblogThemesController.$inject = ['$scope', 'api', '$location', 'notify', 'gettext', '$q', '$sce', 'config', 'lodash', 'upload'];
     function LiveblogThemesController($scope, api, $location, notify, gettext, $q, $sce, config, _, upload) {
+        var vm = this;
```

Comment:
Yes I plan to refactor this controller later in another PR (this one is already too big)
This is really better to use the ControllerAs syntax instead of $scope (see http://toddmotto.com/digging-into-angulars-controller-as-syntax/)
Here, using the controllerAs syntax helps me to don't create a method to close the modal. I can access `vm.themePreviewModal` inside the controller scope, and the modal scope.